File: auto-docs/executables/python/error_bar_asymmetric_constant.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Scatter(
        x=[1, 2, 3, 4],
        y=[2, 1, 3, 4],
        error_y=ErrorY(
            type='percent',
            symmetric=False,
            value=15,
            valueminus=25
        )
    )
])
plot_url = py.plot(data, auto_open=False, filename='error-bar-asymmetric-constant')

File: auto-docs/executables/nodejs/error_bar_asymmetric_constant.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [1, 2, 3, 4], 
    y: [2, 1, 3, 4], 
    error_y: {
      type: "percent", 
      symmetric: false, 
      value: 15, 
      valueminus: 25
    }, 
    type: "scatter"
  }
];

var graph_options = {filename: "error-bar-asymmetric-constant", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});